samples/cpp/benchmark_app/main.cpp

```diff
@@ -191,6 +191,10 @@ int main(int argc, char* argv[]) {
             ie.AddExtension(extension_ptr);
             slog::info << "CPU (MKLDNN) extensions is loaded " << FLAGS_l << slog::endl;
         }
+        // Set throughput when loading AUTO device if hint is empty.
+        if (FLAGS_d.find("AUTO") != std::string::npos && FLAGS_hint.empty()) {
```

Comment:
why this is AUTO specific, thss makes AUTO:CPU and CPU perf incomparable (when no hint is specified).
Instead when all the plugins matured with the hints support, the FLAGS_hint will have a default value of "THROUGHPUT" for any device

Reply:
Updated.